client/src/components/Feed/FiltersList.js

```diff
@@ -0,0 +1,76 @@
+// Core
+import React, { useContext } from "react";
+import styled from "styled-components";
+
+// Antd 
+import { Button } from "antd-mobile";
+
+// Svg
+import { ReactComponent as CloseIcon } from "assets/icons/close-btn.svg";
+
+// Local
+import { FeedContext } from "pages/Feed.js";
+
+// Constants
+import { theme } from "constants/theme";
+import { ROYAL_BLUE, WHITE } from "constants/colors";
+
+const TagButton = styled(Button)`
```

Comment:
I think `FilterTag.js` is the same as this? Maybe we can reuse it?

Reply:
Can we try to get the positioning of the tags like the designs?

current
<img width="256" alt="Screen Shot 2020-05-10 at 8 35 41 PM" src="https://user-images.githubusercontent.com/42228386/81522096-2ea2ef80-92fe-11ea-96e9-6330c144e199.png">

figma
<img width="288" alt="Screen Shot 2020-05-10 at 8 35 37 PM" src="https://user-images.githubusercontent.com/42228386/81522101-3367a380-92fe-11ea-98e7-58c7f8d8eec2.png">

Reply:
Filtertag is based in antd mobile Tag component and it seems that close icon cannot be changed.

Reply:
Tags positioning: done.

Reply:
> Filtertag is based in antd mobile Tag component and it seems that close icon cannot be changed.

Ah good point, we should be semantic with elements. If possible can you replace all the places that are using `FilterTag` to use this instead?

Reply:
I think the tags needs some styling when clicked/active/focused. Right now on click the size of the tag grows and moves surrounding tags:

before clicking:
![Screen Shot 2020-05-11 at 4 44 12 PM](https://user-images.githubusercontent.com/42228386/81624563-5308d580-93ab-11ea-9122-dfa7d07b587d.png)

after clicking:
![Screen Shot 2020-05-11 at 4 44 17 PM](https://user-images.githubusercontent.com/42228386/81624572-58662000-93ab-11ea-9cba-a4b8edfc4c1d.png)

Reply:
Created a new component to be used for tags. All issues fixed.

Reply:
-1 On mobile the tags aren't very responsive to click:
![mobile-tags](https://user-images.githubusercontent.com/42228386/81642341-0a1b4600-93d8-11ea-9c82-2d4d9da7a819.gif)

-2 On mobile there's a slight border when the tag is clicked (can see from above). It's removed by removing the border:
![Screen Shot 2020-05-11 at 10 29 10 PM](https://user-images.githubusercontent.com/42228386/81642425-33d46d00-93d8-11ea-833e-44a307c92e5c.png)

-3 On mobile, the tags in "Need help give help" are different sizes:

current
![Screen Shot 2020-05-11 at 10 41 23 PM](https://user-images.githubusercontent.com/42228386/81642622-a9403d80-93d8-11ea-87ed-6be1b8cf46be.png)

figma
![Screen Shot 2020-05-11 at 10 41 10 PM](https://user-images.githubusercontent.com/42228386/81642620-a8a7a700-93d8-11ea-94ba-5b2d10e5c842.png)

Reply:
It's weird the behavior of antd mobile button in these tags, I will leave filterTag for mobile and Button Tag for desktop. Ok?

Reply:
Ok sounds good, I can try looking into it too.

Reply:
I think I found the issue with the button behavior, I will try with ButtonTag again and if it doesn't work good I will integrate FilterTag.